src/cfapi_client_api.erl

```diff
@@ -44,14 +44,14 @@ get_all_segments(Ctx, EnvironmentUUID, Optional) ->
     Cfg = maps:get(cfg, Optional, application:get_env(kuberl, config, #{})),
 
     Method = get,
-    Path = [<<"/client/env/", EnvironmentUUID, "/target-segments">>],
+    Path = [<<"/client/env/", EnvironmentUUID/binary, "/target-segments">>],
```

Comment:
Not sure why this is showing up in this diff @bmjen - this binary syntax is already in the main branch